Exercise_CIR1_HHQFQDoFile2_2Page_Analysis.do

```diff
@@ -98,11 +98,11 @@ cd "$datadir"
 * Set macros for data sets
 //local householddata: dir . files "*.dta"
 //local householddata: dir . files "CIR1_WealthWeightAll_6Dec2017.dta"
-local householddata "/Users/joeflack4/projects/git-workshop/data/CI/R1/CIR1_WealthWeightAll_6Dec2017.dta"
+local householddata "~/Dropbox (Gates Institute)/git-workshop/data/CI/R1/CIR1_WealthWeightAll_6Dec2017.dta"
```

Comment:
Thanks Suzanne!

Too bad Stata probably doesn't do it exactly the way I wanted (outside of batch mode). You guys should use R! =P =P